src/Nav.jsx

```diff
@@ -0,0 +1,9 @@
+function Nav() {
+  return (
+    <nav>
+      <button className="topicsButton">Topics</button>
```

Comment:
I know these buttons are placeholders - but they should be links if you intend for them to relocate the user